Polyfills/XMLHttpRequest/Source/XMLHttpRequest.cpp

```diff
@@ -212,6 +246,17 @@ namespace Babylon::Polyfills::Internal
             throw Napi::Error::New(info.Env(), "XMLHttpRequest must be opened before it can be sent");
         }
 
+        if (info.Length() > 0)
+        {
+            if (!info[0].IsString() && !info[0].IsUndefined())
+            {
+                throw Napi::Error::New(info.Env(), "Only strings are supported in XMLHttpRequest body");
+            }
+
+            std::string requestBody = info[0].IsString() ? info[0].As<Napi::String>().Utf8Value() : std::string();
+            m_request.SetRequestBody(requestBody);
```

Comment:
Is this what happens on the web (i.e., passing in an undefined will cause the body to be set to empty string)? Also null is not the same as undefined in JS. Probably should check for that also. Looking at the spec, I would expect passing in undefined or null will simply not set the body at all.

Reply:
Reviewed spec and not setting body for undefined and null